packages/dataviews/src/filter-summary.js

```diff
@@ -167,19 +179,81 @@ export default function FilterSummary( {
 				toggleRef.current?.focus();
 			} }
 			renderToggle={ ( { isOpen, onToggle } ) => (
-				<Button
-					__experimentalIsFocusable
-					size="compact"
-					onClick={ onToggle }
-					aria-expanded={ isOpen }
-					ref={ toggleRef }
-				>
-					<FilterText
-						activeElement={ activeElement }
-						filterInView={ filterInView }
-						filter={ filter }
-					/>
-				</Button>
+				<div className="dataviews-filter-summary__chip-container">
+					<Tooltip
+						text={ sprintf(
+							/* translators: 1: Filter name. */
+							__( 'Filter by: %1$s' ),
+							filter.name.toLowerCase()
+						) }
+						placement="top"
+					>
+						<div
+							className={ classnames(
+								'dataviews-filter-summary__chip',
+								{
+									'has-reset': canResetOrRemove,
+									'has-values': hasValues,
+									'is-primary': isPrimary,
+								}
+							) }
+							role="button"
+							tabIndex={ 0 }
+							// eslint-disable-next-line jsx-a11y/no-noninteractive-element-to-interactive-role
+							onClick={ onToggle }
+							onKeyDown={ ( event ) => {
+								if (
+									[ ENTER, SPACE ].includes( event.keyCode )
```

Comment:
This is what happens when the filter chip is focused and I hit space (notice that it works fine in the view actions):

https://github.com/WordPress/gutenberg/assets/583546/f3455d85-b9b1-4809-b3b0-66e639ea5767

Reply:
Thanks for catching this! I fixed it.